<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MW</class>
 <widget class="QMainWindow" name="MW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оптимизация</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lea1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lfun">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>f(x,y) =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lepowx1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lx1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x^</string>
    </property>
   </widget>
   <widget class="QLabel" name="ly1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y^</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lepowy1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lepowy2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lepowx2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ly2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y^</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lea2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lx2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x^</string>
    </property>
   </widget>
   <widget class="QLabel" name="lplus1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="lplus2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lea3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lx3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x^</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lepowy3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ly3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y^</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lepowx3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbexec">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lestatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lstatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Статус</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbgraph">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>График</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>456</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lea1</tabstop>
  <tabstop>lepowx1</tabstop>
  <tabstop>lepowy1</tabstop>
  <tabstop>lea2</tabstop>
  <tabstop>lepowx2</tabstop>
  <tabstop>lepowy2</tabstop>
  <tabstop>lea3</tabstop>
  <tabstop>lepowx3</tabstop>
  <tabstop>lepowy3</tabstop>
  <tabstop>lestatus</tabstop>
  <tabstop>pbexec</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
